ssg/ansible.py

```diff
@@ -22,4 +23,14 @@ def add_minimum_version(ansible_src):
           """ % (ansible_version_requirement_pre_task_name,
                  min_ansible_version, min_ansible_version))
 
+    if ' - hosts: all' not in ansible_src:
```

Comment:
So... We apparently build on older `oscap`s than 1.2.16. 

This appears related to this PR: https://github.com/OpenSCAP/openscap/pull/895

When doing so, we do not include the Ansible snippet, but otherwise do not update the content accordingly.

The `min_ansible_version` is 2.3, which was released ["Mid 2017"](https://docs.ansible.com/ansible/2.5/roadmap/ROADMAP_2_3.html) and tagged pApril 2017](https://github.com/ansible/ansible/releases/tag/v2.3.0.0-1). 

OpenSCAP 1.2.16 was released in [November 2017](https://github.com/OpenSCAP/openscap/releases/tag/1.2.16). 

I'd have thought it to be somewhat unlikely that OpenSCAP 1.2.16 is not included but Ansible 2.3 is... However, Ubuntu proves me wrong as always:

- [Ansible v2.5](https://packages.ubuntu.com/search?suite=bionic&section=all&arch=any&keywords=Ansible&searchon=names)
- [OpenSCAP v1.2.15](https://packages.ubuntu.com/search?keywords=OpenSCAP&searchon=names&suite=bionic&section=all)

So we might want to:

1. Poke the Ubuntu/Debian maintainers for OpenSCAP projects and get them to bump the versions.
2. Figure out what we want to do about minimum Ansible versions including in this.

Until then, I'm changing the `ValueError` to a `return ansible_src`.